tests/integration/Endpoints/PaymentTest.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace Alma\API\Tests\Integration;
+
+use Alma\API\Client;
+use Alma\API\Entities\Instalment;
+use Alma\API\Entities\Payment;
+use PHPUnit\Framework\TestCase;
+
+final class PaymentsTest extends TestCase
+{
+    protected static $almaClient;
+
+    public static function setUpBeforeClass()
+    {
+        self::$almaClient = new Client(
+            $_ENV['ALMA_API_KEY'],
```

Comment:
No API key here ? ;-)

Reply:
I think here it's fine if we remove the other one

Reply:
;-)